tests/lax_numpy_test.py

```diff
@@ -1811,8 +1811,11 @@ def testReshapeMethod(self, arg_shape, out_shape, dtype, rng_factory):
        "rng_factory": jtu.rand_default}
       for arg_shape in [(), (3,), (3, 4)]
       for dtype in default_dtypes
-      for dim in range(-len(arg_shape)+1, len(arg_shape))))
+      for dim in (list(range(-len(arg_shape)+1, len(arg_shape)))
+                  + [(0,), (len(arg_shape), len(arg_shape) + 1)])))
   def testExpandDimsStaticDim(self, arg_shape, dtype, dim, rng_factory):
+    if isinstance(dim, tuple) and numpy_version < (1, 18, 0):
```

Comment:
I think you only need to skip the `CheckAgainstNumpy` test, but you can leave the `CompileAndCheck` part of the test enabled.